examples/longbench/README.md

```
@@ -0,0 +1,170 @@
+
+Install the required dependencies:
+
+```bash
+pip install -r requirements.txt
+```
+
+### 3. Directory Structure
+
+After cloning, your directory structure should look like:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)